config.toml

```diff
@@ -111,7 +111,8 @@ white_color = "#ffffff"
 light_color = "#f8f9fa"
 
 # font family (choose from https://fonts.google.com/)
-font_family = "Open+Sans"
+font_family_a = "Open"
+font_family_b = "Sans"
```

Comment:
Not sure I understand this. So, the font I was using is the [Open Sans](https://fonts.google.com/specimen/Open+Sans?query=Open+Sans) font. Could you please explain this change to me, or why it is necessary?

Reply:
@jwflory  I did that to avoid the error that pops up in the console
![inventory error](https://user-images.githubusercontent.com/56026636/116749893-8350e600-a9f9-11eb-821a-fa4590b6959f.png)

When I made that change I went into the themes/inventory/layouts/partials/head.html file and made some changes. I changed from this
```
  {{ "<!-- fonts -->" | safeHTML }}
  <link href="https://fonts.googleapis.com/css?family={{replace .Site.Params.font_family ' ' '+' | title}}:300,400,700&display=swap" rel="stylesheet">
```
To this:
```
  {{ "<!-- fonts -->" | safeHTML }}
  <link rel="preconnect" href="https://fonts.gstatic.com">
  <link href="https://fonts.googleapis.com/css2?family={{replace .Site.Params.font_family_a ' ' '+' | title}}+{{replace .Site.Params.font_family_b ' ' '+' | title}}:wght@300;400;700&display=swap" rel="stylesheet"> 
```

And then that error disappears...